src/lexer.coffee

```diff
@@ -48,6 +48,7 @@ exports.Lexer = class Lexer
     @seenExport = no             # Used to recognize EXPORT FROM? AS? tokens.
     @importSpecifierList = no    # Used to identify when in an IMPORT {...} FROM? ...
     @exportSpecifierList = no    # Used to identify when in an EXPORT {...} FROM? ...
+    @includesCSX = no            # Used to optimize CSX checks.
```

Comment:
So this is just a flag that gets set to true the first time a CSX tag is encountered? How are we saving any processing, if we have to check for CSX tags in order to determine that we can skip CSX processing?